libsolc/libsolc.cpp

```diff
@@ -73,9 +73,9 @@ ReadCallback::Callback wrapReadCallback(CStyleReadFileCallback _readCallback = n
 	return readCallback;
 }
 
-string compile(string _input, CStyleReadFileCallback _readCallback = nullptr)
+string compile(string _input, CStyleReadFileCallback _readCallback, void* _readContext)
```

Comment:
Maybe make both default `= nullptr`?

Reply:
This is an internal helper within the file. It is only used once by the external wrapper `solidity_compile`. No need for defaults.

The reason the helper exists because earlier we had multiple entry points, but even now it doesn't seem to hurt readability.